lib/betagouvbot/account_request.rb

```diff
@@ -0,0 +1,73 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module BetaGouvBot
+  module AccountRequest
+    module_function
+
+    InvalidNameError   = Class.new(StandardError)
+    InvalidEmailError  = Class.new(StandardError)
+    EmptyPasswordError = Class.new(StandardError)
+
+    class << self
+      def call(authors, params)
+        fullname, email, password = params.split
```

Comment:
Sure. But I have one problem : there's no integration test for webhook... should I add it here for this PR ?